_posts/2024-05-23-pyos-pyconus-2024-recap.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: single
+title: "pyOpenSci at PyCon US 2024 - Python packaging and community"
+excerpt: "Learn about  pyOpenSci's experience at PyCon US 2024, how pyOpenSci is making Python packaging more accessible and beginner friendly and how the Python community is helping us get there."
+author: "Leah Wasser"
+permalink: /blog/recap-pyos-pyconus-2024.html
+header:
+  overlay_image: images/blog/2024/may/pycon-us-2024.png
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+
+<!-- TODOs:
+* embed video when live
+* add graphics from social that jesse sent
+* fix captions
+* add stravalib note at the end of my talk section -->
+
+
+<div class="notice" markdown = "1">
+## TL&DR
+
+* pyOpenSci has a strong presence at pyCon US this year. I hope this continues for years to come! We held an open space, helped run the maintainers summit (lead by Inessa Pawson for 5 years and counting), gave a talk and ran a 1 day sprint where over 16 people contributed to our efforts.
+* pyOpenSci’s theme this year for PyConUS was people first. People first when trying to make technical concepts easier to understand, people first when trying to write good tutorials or documentation and people first when you are trying to solve the world’s hardest problems.
+* Giving a talk on packaging at pyConUS triggered every ounce of the imposter in me. But in the end it was a rewarding experience. Having friends in the audience made a world of difference. It was calming to focus on people who I know support both me and this vibrant organization. Friends really should never let friends … package … or use Python … or do anything technical…  alone.
+</div>
+
+
+## Another year, another pyCon
+
+Wow! I wasn’t sure it would be possible to top last year’s PyCon US 2023 experience
+in Salt Lake City, but this year's [PyConUS 2024](https://us.pycon.org/2024/) *was even better*. People are learning about pyOpenSci. The community is excited to support our mission to **help
+scientists create better software and make their science more open and
+reproducible** so they can tackle the world's greatest challenges.
+
+PyConUS, run by the [Python Software
+Foundation,](https://www.python.org/psf-landing/) is one of the biggest Python meetings in the world with a record 2,700 registrations this year.
+{: .notice }
+
+
+I also knew that I'd get to see a bunch of the friends who I met last year. I was returning to
+this inclusive community, filled with Pythonistas like me
+who care, who want to learn, and most importantly who want to help each other.
+
+And... Thanks to our incredible community manager,
+[Jesse (@kiersi on GitHub)](https://github.com/kiersi), we now have
+a [pyOpenSci](https://www.pyopensci.org) visual brand -- and stickers galore.
```

Comment:
YAY STICKERS!

Reply:
except you don't have any yet!! we need to fix that. and beautiful post cards too. but the stickers were popular. i think colored pencils for the sprint for the post cards are a must for scipy. just one set that people can use if they wish to take breaks from sprinting.